superset-frontend/src/views/CRUD/welcome/EmptyState.tsx

```diff
@@ -151,10 +151,9 @@ export default function EmptyState({ tableName, tab }: EmptyStateProps) {
             window.location.href = favRedirects[tableName];
           }}
         >
-          See all{' '}
-          {tableName === 'SAVED_QUERIES'
-            ? t('SQL Lab queries')
-            : t(`${tableName}`)}
+          {t('See all %(tableName)s', {
+              tableName: welcomeTableLabels[tableName],
```

Comment:
I think I may have read this too hastily the first time. I think something like this should mimic the original intent:
```ts
t('See all %(tableName)s', { tableName: tableName === 'SAVED_QUERIES' ? t('SQL Lab queries') : welcomeTableLabels[tableName] }
```